frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -126,6 +126,8 @@ interface ITableContainerProps<T = any> {
 }
 
 const baseClass = "table-container";
+/** Styling reused in software edit icon preview modal */
+export const resultsCountClass = "table-container__results-count";
```

Comment:
if we are are only using this in one place can we handle the class and styles there? or do you anticipate other places will need to use this class in the future?

Reply:
Yeah, for this, it's silly because design has the preview look like the actual page, but I'm worried if the page changes looks (which design plans to do a styling overhaul in Q2), that this would get outdated too. This was my only reasonable thought process to keep styling consistent, wdyt?

Reply:
I updated this, lmkwyt!